deepchem/molnet/run_benchmark.py

```diff
@@ -209,3 +209,100 @@ def run_benchmark(datasets,
         output_line.extend(
             ['time_for_running', time_finish_fitting - time_start_fitting])
         writer.writerow(output_line)
+
+
+#
+# Note by @XericZephyr. Reason why I spun off this function:
+#   1. Some model needs dataset information.
+#   2. It offers us possibility to **cache** the dataset
+#      if the featurizer runs very slow, e.g., GraphConv.
+#   2+. The cache can even happen at Travis CI to accelerate
+#       CI testing.
+#
+def load_dataset(dataset,
+                 split=None,
+                 featurizer=None):
```

Comment:
I suggest to change the `featurizer` to a positional argument in this function, as `None` is not accepted in most dataset loading functions